lib/network_ssl/network_ssl.c

```diff
@@ -60,6 +62,34 @@ freectx(void)
 	ctx = NULL;
 }
 
+static const char * common_certfiles[] = {
+	"/usr/local/share/certs/ca-root-nss.crt",
+	"/etc/ssl/certs/ca-certificates.crt"
+};
+static const size_t num_common_certfiles = sizeof(common_certfiles) /
+    sizeof(common_certfiles[0]);
+
+/**
+ * network_ssl_default_certfile():
+ * Look for root certificates in a few common places in the filesystem.  If
+ * a non-NULL value is returned, it can probably be used with
+ * network_ssl_loadroot.
+ */
+const char *
+network_ssl_default_certfile()
```

Comment:
What are your thoughts about something like this?  As per the comment (which isn't as noticeable in github):

I think it's good to have a function like there somewhere in kivaloo.  You probably won't like <sys/stat.h> in lib/network_ssl, though.  I'm happy to move it elsewhere -- lib/certfiles ?

Reply:
Hmm.  I'm not a fan of having library code groping through the filesystem looking for a file.  If there was one consistent place where everyone installed root certificates, it would be marginally ok, but if there's a bunch of different places to look...

Wait, I'm an idiot.  OpenSSL already provides `SSL_CTX_set_default_verify_paths`, which loads root certificates from whatever the appropriate place is on that platform.  So let's just use that.  (And `network_ssl_loadroot` can go away and this can all be done as part of `init()`.)